Documentation/cli-installation-scenarios.md

```diff
@@ -0,0 +1,216 @@
+Obtaining .NET CLI 
+==================
+
+# Contents
+* [Overview](#overview)
+* [General principles](#general-principles)
+* [Channels](#channels)
+* [Funnels and discovery mechanisms for CLI bits](#funnels-and-discovery-mechanisms-for-cli-bits)
+  * [Getting Started page](#getting-started-page)
+  * [Repo landing page](#repo-landing-page)
+* [Dependencies](#dependencies)
+* [Upgrades](#upgrades)
+* [Layout on disk](#layout-on-disk)
+* [Acquisition modes](#acquisition-modes)
+  * [Native installers](#native-installers)
+  * [Installation script](#installation-script)
+    * [Windows one-liner](#windows-command)
+    * [OSX/Linux one-liner](#osxlinux-shell-command)
+  * [Complete manual installation](#complete-manual-installation)
+  * [Docker](#docker)
+  * [NuGet Packages](#nuget-packages)
+* [Acquiring through other products](#acquiring-through-other-products)
+  * [IDEs and editors](#ides-and-editors)
+  
+
+# Overview
+This document/spec outlines the CLI install experience. This covers the technologies being used for install, the principles that are driving the installation experience, the ways users are coming to the installs and what each of the installs contains, in terms of stability and similar. 
+
+# General principles
+
+- Upgrades using the native installers Just Work(tm)
+- All user facing materials point to the getting started page
+- Defaults are stable bits; users need extra effort to install nightly builds
+- Only HTTPS links are allowed in any online property 
+- Provide native installers for each supported platform
+- Provide automation-ready installers for each target platform
+
+# Channels
+Channels represent a way for users who are getting the CLI to reason about the stability and quality of the bits they are getting. This is one more way for the user to be fully aware of the state the bits that are being installed are in and to set proper expectations on first use. 
+
+The table below outlines the channels:
+
+| Property         	| Description                                                                                                                                                                                                                                                       	|
+|------------------	|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------	|
+| Nightly              	| Unstable bits that are "bleeding edge". Users are not expected to use this channel often, however it is there for those situations when someone needs/wants a feature that hasn't been stabilizied yet. Also, used for internal testing. 	|
```

Comment:
since this is used in the scripts, we should decide on capitalization or make a specific decision on how this string should look like in the scripts